app/tests/test_routes.py

```diff
@@ -97,14 +97,13 @@ def create_mock_s3_bucket_with_image_object(bucket_name, file):
     image_file = BytesIO()
     image = Image.new("RGB", (800, 600), color=(73, 109, 137))
     # Determine format from file extension
```

Comment:
```suggestion
    # Determine format from file puid
```

or just remove the comment

Reply:
comment removed